lib/dataContract/stateTransition/DataContractCreateTransition.js

```diff
@@ -60,20 +64,54 @@ class DataContractCreateTransition extends AbstractStateTransitionIdentitySigned
    *
    * @param {Object} [options]
    * @param {boolean} [options.skipSignature]
-   *
+   * @param {boolean} [options.encodedBuffer=false]
    * @return {Object}
    */
   toObject(options = {}) {
-    return {
+    Object.assign(
+      options,
+      {
+        encodedBuffer: false,
+        ...options,
+      },
+    );
+
+    let plainObject = {
       ...super.toObject(options),
-      dataContract: this.getDataContract().toJSON(),
+      dataContract: this.getDataContract().toObject(),
```

Comment:
shouldn't we pass options to `DataContract#toObject` too?

Reply:
it doesn't accept any option right now.